src/main/java/study/racing/domain/Car.java

```diff
@@ -1,33 +1,26 @@
 package study.racing.domain;
 
 import study.racing.domain.move.MoveStrategy;
-import study.racing.domain.move.RandomMoveStrategy;
 
 import java.util.Objects;
 
 public class Car {
 
     private final CarName name;
     private CarPosition position;
-    private final MoveStrategy moveStrategy;
 
     public Car(String name) {
-        this(new CarName(name), new CarPosition(0), new RandomMoveStrategy());
+        this(new CarName(name), new CarPosition(0));
```

Comment:
```suggestion
        this(new CarName(name), new CarPosition());
```
CarPosition에 기본 생성자(0을 초기화함)를 추가하면 위와 같이 구현 가능함.
객체에 여러 생성자, 메서드를 추가해 이 객체를 사용하는 개발자가 하드 코딩을 줄이고, 최대한 사용의 편리성을 가지도록 설계하는 습관을 들이면 어떨가요?